Chakra migration guide - add breakpoints guidance (#7829)
ethereum/ethereum-org-website: docs/chakra-migration-guide.md

<Text fontSize="md" margin={4} />
```

## Breakpoints

> We are going to use [the Chakra default breakpoints](https://chakra-ui.com/docs/styled-system/theme#breakpoints) from now on. Check the following table to do the conversion:

Reviewer: ```suggestion
We will use [the Chakra default breakpoints](https://chakra-ui.com/docs/styled-system/theme#breakpoints) from now on. Check the following table to do the conversion:
```

> | old breakpoints | new breakpoints |
> |---|---|
> | xs | - |
> | s | sm |
> | m | md |
> | l | lg |
> | xl | xl |
> | - | 2xl |

Reviewer: Just making this a little tidier.

```suggestion
| old breakpoints | new breakpoints |
|-----------------|-----------------|
| xs              | -               |
| s               | sm              |
| m               | md              |
| l               | lg              |
| xl              | xl              |
| -               | 2xl             |

```
